README.md

```diff
@@ -12,11 +12,29 @@
 
 </div>
 
-This workspace is a collection of constants and tools for interoperating US Customary and SI units of length/displacement, with a focus on avoiding unnecessary loss of precision and idempotency.
+## Operating principle
 
-In the `joto_constants` package, `IOTA` is defined as one ninth of a nanometer.
-This allows common fractions of an inch (ten-thousandths, desktop publishing points, and sixty-fourths) and nanometers to be represented as integers.
-Using this base unit, combinations of lengths in either [US customary units](<https://en.wikipedia.org/wiki/United_States_customary_units>) or [SI units](<https://en.wikipedia.org/wiki/International_System_of_Units>) can be added, subtracted, and multiplied without loss of precision.
+This workspace is a collection of constants and tools for interoperating US Customary and SI units of length/displacement, mass, and temperature, with a focus on preserving invertibility, avoiding uncontrolled loss of precision, and replacing structured unit types (which are hard to store or compute efficiently) with plain integers.
+
+This is accomplished by defining common base units for length, mass, and temperature, which can be used to express both [US customary units](<https://en.wikipedia.org/wiki/United_States_customary_units>) and [SI units](<https://en.wikipedia.org/wiki/International_System_of_Units>) in the same scale.
+
+### Length/displacement
+
+For length, there is the *iota*, defined as one ninth of a nanometer.
+
+This allows common fractions of an inch (ten-thousandths, desktop publishing points, and sixty-fourths) and multiples of the nanometer to be represented as natural numbers.
+
+### Mass
+
+For mass, there is the *whit*, defined as 1⁄3200 µg.
+
+The whit is chosen to express practically measurable weights (down to the 0.1 µg range) as well as all common fractional denominations of the international pound (ounces, dram, thousandths of an ounce, grains) and units related to the pound by grains (such as the troy ounce).
+
+### Temperature
+
+For temperature, there is the *smidge*, defined as 1⁄90 mK.
+
+The smidge represents temperatures down to the 100 µK/0.0001 °R range, which is sufficient for almost all practical thermometry. This also allows you to exactly represent temperatures used in industrial metrology standards for fixed points and common derived constants (e.g. the triple point or the two-point calibrated boiling point of water), and allows exact interchange between common absolute (Kelvin/Rankine) and relative (Celsius/Fahrenheit) temperature scales.
```

Comment:
Assuming the standard you mean here is ITS-90, among others? Would be good to link it. Might also be good to link Rankine as I don't think that's general knowledge.